src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -2409,7 +2409,10 @@
     text: az vmss update --name MyScaleSet --resource-group MyResourceGroup --instance-id 4 --protect-from-scale-set-actions False --protect-from-scale-in
   - name: Update a VM instance's protection policies.
     text: az vmss update --name MyScaleSet --resource-group MyResourceGroup --instance-id 4 --set protectionPolicy.protectFromScaleIn=True protectionPolicy.protectFromScaleSetActions=False
-
+  - name: Update a VM instance's Read-Write IOPS of the managed disk.
+    text: az vmss update --name MyScaleSet --resource-group MyResourceGroup --set virtualMachineProfile.storageProfile.dataDisks[0].diskIOPSReadWrite=444
+  - name: Update a VM instance's bandwidth in MB per second of the managed disk.
+    text: az vmss update --name MyScaleSet --resource-group MyResourceGroup --set virtualMachineProfile.storageProfile.dataDisks[0].diskMBpsReadWrite=66
```

Comment:
Looks like these example commands can use `--data-disk-iops`, `--data-disk-mbps` instead of generic update.

Reply:
It's not supported in `update` now.